styles/colours/colourpalette.md

```diff
@@ -3,61 +3,113 @@ description: Primary and secondary colours to be used on all government digital
-![#5475a7](https://github.com/bcgov/design-system/blob/master/styles/colours/images/5475a7.png?raw=true) | #5475a7 | 84 117 167 | Second level background colour for navigation bar
-![#f2f2f2](https://github.com/bcgov/design-system/blob/master/styles/colours/images/f2f2f2.png?raw=true) | #f2f2f2 | 242 242 242 | Background grey left and right side bars
-![#fcba19](https://github.com/bcgov/design-system/blob/master/styles/colours/images/fcba19.png?raw=true) | #fcba19 | 252 186 25 | Highlight gold bar under header and above footer
-![#494949](https://github.com/bcgov/design-system/blob/master/styles/colours/images/494949.png?raw=true) | #494949 | 737373 | Site wide text on white background
-![#bcbec5](https://github.com/bcgov/design-system/blob/master/styles/colours/images/bcbec5.png?raw=true) | #bcbec5 | 188 190 197 | Return to op icon, share button icon, search icon, search box text and search box border
-
-## Secondary Colours
-Visual Example | Hex Code | RGB | Example Use
------------- | ------------ | ------------ | ------------ |
-![#d0d0d1](https://github.com/bcgov/design-system/blob/master/styles/colours/images/d0d0d1.png?raw=true) | #d0d0d1 | 208 208 209 | ???
-![#afafaf](https://github.com/bcgov/design-system/blob/master/styles/colours/images/afafaf.png?raw=true) | #afafaf | 175 175 175 | ???
-![#616265](https://github.com/bcgov/design-system/blob/master/styles/colours/images/616265.png?raw=true) | #616265 | 97 98 191 | ???
-![#8a9198](https://github.com/bcgov/design-system/blob/master/styles/colours/images/8a9198.png?raw=true) | #8a9198 | 138 145 152 | ???
-![#002663](https://github.com/bcgov/design-system/blob/master/styles/colours/images/002663.png?raw=true) | #002663 | 0 38 99 | ???
-![#5091cd](https://github.com/bcgov/design-system/blob/master/styles/colours/images/5091cd.png?raw=true) | #5091cd | 80 145 205 | ???
-![#96c0e6](https://github.com/bcgov/design-system/blob/master/styles/colours/images/96c0e6.png?raw=true) | #96c0e6 | 150 192 230 | ???
-![#f3bd48](https://github.com/bcgov/design-system/blob/master/styles/colours/images/f3bd48.png?raw=true) | #f3bd48 | 243 189 72 | ???
-![#f3cd65](https://github.com/bcgov/design-system/blob/master/styles/colours/images/f3cd65.png?raw=true) | #f3cd65 | 243 203 101 | ???
-![#fade81](https://github.com/bcgov/design-system/blob/master/styles/colours/images/fade81.png?raw=true) | #fade81 | 250 222 129 | ???
-![#486446](https://github.com/bcgov/design-system/blob/master/styles/colours/images/486446.png?raw=true) | #486446 | 72 100 70 | ???
-![#481124](https://github.com/bcgov/design-system/blob/master/styles/colours/images/481124.png?raw=true) | #481124 | 72 33 36 | ???
-![#d8292f](https://github.com/bcgov/design-system/blob/master/styles/colours/images/d8292f.png?raw=true) | #d8292f | 216 41 47 | ???
-![#a12622](https://github.com/bcgov/design-system/blob/master/styles/colours/images/a12622.png?raw=true) | #a12622 | 161 38 34 | ???
-
-## Rationale
```

Comment:
The US design system has a little paragraph around the meaning of the colours. Might be nice to keep but also can be wordy and I am ok removing it.

Reply:
Let's keep it simple for now and then add in more content